src/web/views/page-not-found.njk

```diff
@@ -0,0 +1,17 @@
+{% extends "templates/page.njk" %}
+
+{% set mainClasses = "govuk-main-wrapper--l" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">{{ heading }}</h1>
+        <p class="govuk-body">
+          {{ checkTypedAddress }}
+        </p>
+        <p class="govuk-body">
+          {{ checkPastedAddress }}
```

Comment:
Should we be using locale based HTML partials here instead of passing strings for each paragraph to the view?